hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerMetadataInspector.java

```diff
@@ -525,12 +526,12 @@ static PendingDelete countPendingDeletesSchemaV2(
 
   static long computePendingDeleteBytes(List<Long> localIDs,
       KeyValueContainerData containerData,
-      Table<String, BlockData> blockDataTable) {
+      Table<String, BlockData> blockDataTable, DatanodeStoreWithIncrementalChunkList schemaTwoStore) {
```

Comment:
The parameter blockDataTable is not used in the function now. Maybe we can clean it up.